docs/server/source/events/websocket-event-stream-api.rst

```diff
@@ -94,11 +94,11 @@ Example message:
     {
         "transaction_id": "<sha3-256 hash>",
         "asset_id": "<sha3-256 hash>",
-        "block_id": "<sha3-256 hash>"
+        "block_id": "<string>"
```

Comment:
@vrde - @kansi has changed the `block_id` type from `string` to `int` in pull request #2167 , so maybe you could change this line in the docs _in this pull request_ and then we will be done with that issue (issue #2166).